templates/verify.html

```diff
@@ -1,31 +1,124 @@
+    <link rel="stylesheet" href="{{ url_for('static', filename='css/flexboxgrid-helpers.min.css') }}">
+
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+        <!--[if lt IE 9]>
+          <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+          <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+          <![endif]-->
+
+        <!-- These two JS are loaded at the top for gray scale including the loader. -->
+
+        <script src="{{ url_for('static', filename='js/jquery-1.11.3.min.js') }}"></script>
+        <!-- jQuery (https://jquery.com/download/) -->
+
+        <link rel="stylesheet" href="{{ url_for('static', filename='style1.css') }}">
+        <link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+        <script src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+        <script>
+        $(function() {
+            $( ".dtpick" ).datepicker();
+        });
+        </script>
+
+        <script>
+		
+            var tm_gray_site = false;
+            
+            if(tm_gray_site) {
+                $('html').addClass('gray');
+            }
+            else {
+                $('html').removeClass('gray');   
+            }
+        </script>
+</head>
+
+<body>
+    {% with messages = get_flashed_messages() %}
+        {% if messages %}
+            {% for message in messages %}
+            <div class="alert">
+                {{ message }}
+            </div>
+            {% endfor %}
+        {% endif %}
+    {% endwith %}
+    <div class="cd-hero">
+        <!-- Navigation -->        
+        <div class="cd-slider-nav">
+            <nav class="navbar">
+                <div class="tm-navbar-bg">     
+                    <a class="navbar-brand text-uppercase" href="{{ url_for('root') }}"><i class="fa fa-paw tm-brand-icon"></i>Pet Adoption</a>              
+                </div>
+
+            </nav>
+        </div>
+        <ul class="cd-hero-slider"> git
+            <li class="selected">
+                <div class="cd-full-width">
+                    <div class="container-fluid js-tm-page-content tm-page-pad" data-page-no="1">
+                        <div class="tm-contact-page"> 
+                            <div class="row">
+                                <div class="col-xs-12">
+                                    <div class="tm-flex tm-contact-container">  
+                                        <div class="tm-bg-white-translucent text-xs-left tm-textbox tm-2-col-textbox-2 tm-textbox-padding tm-textbox-padding-contact">
+                                            <h2 class="tm-contact-info">Resend Verification Link</h2>
+                                                    <form action="/login" method="POST" enctype="multipart/form-data" style="box-shadow: none;">
```

Comment:
enctype should not be multipart/form-data unless this form is for uploading files: 

https://flask-wtf.readthedocs.io/en/stable/form.html#module-flask_wtf.file